WMFComponents/Sources/WMFComponents/Components/Year in Review/WMFSlideShow.swift

```diff
@@ -0,0 +1,138 @@
+import SwiftUI
+
+public struct WMFSlideShow: View {
+    @ObservedObject var appEnvironment = WMFAppEnvironment.current
+    @Binding private var currentSlide: Int
+    
+    var theme: WMFTheme {
+        return appEnvironment.theme
+    }
+    
+    let slides: [YearInReviewSlide]
+    
+    public init(currentSlide: Binding<Int>, slides: [YearInReviewSlide]) {
+        self._currentSlide = currentSlide
+        self.slides = slides
+        
+        UIPageControl.appearance().currentPageIndicatorTintColor = theme.link.withAlphaComponent(0.3)
+        UIPageControl.appearance().pageIndicatorTintColor = theme.link.withAlphaComponent(0.3)
+    }
+    
+    public var body: some View {
+        VStack {
+            TabView(selection: $currentSlide) {
+                ForEach(0..<slides.count, id: \.self) { slide in
+                    VStack(alignment: .leading, spacing: 16) {
+                        Image(slides[slide].imageName, bundle: .module)
+                            .frame(maxWidth: .infinity, alignment: .center)
+                        Text(slides[slide].title)
+                            .font(Font(WMFFont.for(.boldTitle1)))
+                            .foregroundStyle(Color(uiColor: theme.text))
+                        Text(slides[slide].subtitle)
+                            .font(Font(WMFFont.for(.title3)))
+                            .foregroundStyle(Color(uiColor: theme.text))
+                        Spacer()
```

Comment:
We might need to wrap this up in a scroll view for larger font sizes so the text isn't cropped.

<img src="https://github.com/user-attachments/assets/e166800a-9c8e-497f-b1a0-95566ba279a3" width="180" height="400">